_pkgdown.yml

```diff
@@ -19,37 +19,12 @@ development:
 
 reference:
 
-- title: Derivations for Adding Variables
-- subtitle: ADXX-specific
-  desc: Derivation Functions helpful for building the ADXX dataset
-- contents:
-  - has_keyword("der_adxx")
-
 - title: Derivations for Adding Parameters
 - subtitle: ADVS-specific
   desc: Parameter Derivation Functions helpful for building the ADVS dataset
 - contents:
   - has_keyword("der_prm_advs")
 
-- title: Advanced Functions
-- subtitle: Pre-Defined Source Objects
-  desc: Source objects defined by {admiralmetabolic}
-- contents:
-  - has_keyword("source_specifications")
-
-- title: Utility Functions
-- subtitle: Utilities for Formatting Observations
-- contents:
-  - has_keyword('utils_fmt')
-
-- subtitle: Utilities for Dataset Checking
-- contents:
-  - has_keyword('utils_ds_chk')
-
-- subtitle: Utilities for Filtering Observations
-- contents:
-  - has_keyword('utils_fil')
-
 - title: Example Datasets
   desc: You can run `admiral::use_ad_template()` to produce additional datasets
```

Comment:
```suggestion
```
I don't see why we need this reference to admiral base template programs. If we want it, I think we should specify how it can be used. For instance, "You can access additional template programs from the admiral package with admiral::use_ad_template().